src/components/Editor/Invitees/InviteesListSearch.vue

```diff
@@ -220,7 +220,7 @@ export default {
 					return false
 				}
 
-				if (this.alreadyInvitedEmails.includes(principal.email)) {
+				if (this.alreadyInvitedEmails.includes(principal.email) && email) {
```

Comment:
> 'email' is not defined

eslint is right :)

Reply:
Exactly, a stupid mistake.

Reply:
It seems like you reverted your change.

Reply:
I added wrong place first time, i reverted it and add new code at 189 line of InviteesListSearch.vue